src/components/HeatMap/Cell/Cell.js

```diff
@@ -1,15 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { setActiveCell, setHourPosts } from '../../../redux/actions/actions';
 import { Wrapper } from './Cell.styles';
 
 
-export default function Cell({
-  itemsNumber, weekDay, hour, activeCell, onCellClick,
+function Cell({
+// eslint-disable-next-line no-shadow
+  itemsNumber, weekDay, hour, activeCell, setActiveCell, hourData, setHourPosts,
 }) {
   const isActive = activeCell[0] === weekDay && activeCell[1] === hour;
 
   const handleClick = () => {
-    onCellClick(weekDay, hour);
+    setActiveCell([weekDay, hour]);
```

Comment:
I think it would be better to use an object for `activeCell`. Accessing the selected day like this `activeCell[0]` is not very descriptive. If I was new to the application I wouldn't know what that is. But something like `activeCell.weekday` is clear